data/json/overmap/overmap_terrain/overmap_terrain_militia.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "type": "overmap_terrain",
+    "id": [
+      "GMLODGE1_1",
+      "GMLODGE1_2",
+      "GMLODGE1_3",
+      "GMLODGE1_4",
+      "GMLODGE2_1",
+      "GMLODGE2_2",
+      "GMLODGE2_3",
+      "GMLODGE2_4",
+      "GMLODGE3_1",
+      "GMLODGE3_2",
+      "GMLODGE3_3",
+      "GMLODGE3_4",
+      "GMLODGE4_1",
+      "GMLODGE4_2",
+      "GMLODGE4_3",
+      "GMLODGE4_4",
+      "GMLODGE2_2_2",
+      "GMLODGE2_3_2",
+      "GMLODGE3_2_2",
+      "GMLODGE3_3_2",
+      "GMLODGE2_roof_2",
+      "GMLODGE2_roof_3",
+      "GMLODGE3_roof_2",
+      "GMLODGE3_roof_3"
+    ],
+    "name": "Militia Lodge",
+    "sym": "L",
+    "color": "blue",
```

Comment:
Blue `L`s are labs, maybe just brown given it's a wooden lodge?
If the surrounding OMTs are staying they should be added to the special_field or special_forest array otherwise it's going to appear way bigger than it is.